### ISSUE TITLE
Train Loss: nan 

### ISSUE THREAD

**Issue by samanemami** [lyp22/ResSCNN#7]
All the epochs of training return Nan as the training loss.
I checked it and it is not coming from the nan gradient.

Still not sure what causes this issue!

```Log
2022-12-06 12:43:12,673 - /home/ResSCNN/lib/Logging.py[line:25] - INFO: Final Loss: 3.499, 
2022-12-06 12:43:12,674 - /home/ResSCNN/lib/Logging.py[line:25] - INFO: Test Loss: 3.499211 
2022-12-06 12:43:12,701 - /home/ResSCNN/lib/Logging.py[line:25] - INFO: PLCC: 0.997658, SROCC: 0.900000,  KROCC: 0.800000,  RMSE: 0.000064, 
2022-12-06 12:43:12,716 - /home/ResSCNN/lib/Logging.py[line:25] - INFO:  Epoch: 1, LR: [0.001]
2022-12-06 12:43:12,951 - /home/ResSCNN/lib/Logging.py[line:25] - INFO: Train Epoch: 1 , Train Loss: nan 
2022-12-06 12:43:32,335 - /home/ResSCNN/lib/Logging.py[line:25] - INFO: Final Loss: 3.499, 
2022-12-06 12:43:32,336 - /home/ResSCNN/lib/Logging.py[line:25] - INFO: Test Loss: 3.499211 
2022-12-06 12:43:32,358 - /home/ResSCNN/lib/Logging.py[line:25] - INFO: PLCC: 0.997658, SROCC: 0.900000,  KROCC: 0.800000,  RMSE: 0.000064, 
2022-12-06 12:43:32,383 - /home/ResSCNN/lib/Logging.py[line:25] - INFO:  Epoch: 2, LR: [0.0009801]
2022-12-06 12:43:32,648 - /home/ResSCNN/lib/Logging.py[line:25] - INFO: Train Epoch: 2 , Train Loss: nan 
2022-12-06 12:43:52,009 - /home/ResSCNN/lib/Logging.py[line:25] - INFO: Final Loss: 3.499, 
2022-12-06 12:43:52,012 - /home/ResSCNN/lib/Logging.py[line:25] - INFO: Test Loss: 3.499211 
2022-12-06 12:43:52,035 - /home/ResSCNN/lib/Logging.py[line:25] - INFO: PLCC: 0.997658, SROCC: 0.900000,  KROCC: 0.800000,  RMSE: 0.000064, 
```

**Comment by lyp22** [comment]
You can try to reduce the learning rate 0.001 -> 0.0001.
Besides, I think that there may also be problems with dataset settings. In almost any case, PLCC and SROCC cannot be over 0.9 at epoch 0 or epoch1.
